### ISSUE TITLE
Reversion data for basically everything needs to be fixed

### ISSUE THREAD

**Issue by ptgolden** [editorsnotes/editorsnotes-server#169]
ugh I mean it's awful.

This is a consequence of all the data migrations I've had to do in significantly changing the site's architecture. It's recoverable but it's awful. We need to address this issue before we can let people browse past versions of content.

**Comment by ptgolden** [comment]
At this point, I'm ready to set this as a wontfix. We *do* need to write tests for reversion data for future items, and we will keep the old data around, but we don't have the time required to massage everything.

We should 100% solidify a basic data model. As in, :100: :100: :100:

**Comment by rybesh** [comment]
I'm fine with wontfix. None of our EN users are going to want to go back through their version history.
